lib/web/integrations_awsoidc.go

```diff
@@ -440,3 +440,61 @@ func (h *Handler) awsOIDCDeployEC2ICE(w http.ResponseWriter, r *http.Request, p
 		Name: resp.Name,
 	}, nil
 }
+
+// awsOIDCConfigureAWSOIDCIdP returns a script that configures the AWS OIDC Integration in AWS.
+// This creates an OIDC Identity Provider that trusts this Teleport instance.
+func (h *Handler) awsOIDCConfigureAWSOIDCIdP(w http.ResponseWriter, r *http.Request, p httprouter.Params) (any, error) {
```

Comment:
```suggestion
func (h *Handler) awsOIDCIdpConfigScript(w http.ResponseWriter, r *http.Request, p httprouter.Params) (any, error) {
```
AWSOIDC reads redundant as the name starts with it?